v2/pink-sb/src/lib/combobox/Combobox.svelte

```diff
@@ -0,0 +1,128 @@
+<script lang="ts">
+    import type { ComboboxOption, ComboboxProps } from '$lib/combobox/index.js';
+
+    export let placeholder: ComboboxProps['placeholder'] = '';
+    export let disabled: ComboboxProps['disabled'] = false;
+    export let options: ComboboxProps['options'] = [];
+    export let label: ComboboxProps['label'] = '';
+
```

Comment:
We might need the `required` state like for other inputs

Reply:
Added $$restProps to the input

Reply:
Not sure if the $$restProps is enough... I think we might need to show a `*` on the label too when it's required (double check with Caio)